java/ql/lib/ext/java.io.model.yml

```diff
@@ -116,6 +116,7 @@ extensions:
       - ["java.io", "File", "isDirectory", "()", "summary", "manual"]
       - ["java.io", "File", "mkdirs", "()", "summary", "manual"]
       - ["java.io", "FileInputStream", "FileInputStream", "(File)", "summary", "manual"]
+      - ["java.io", "InputStream", "read", "()", "summary", "manual"]
```

Comment:
Shouldn't this be a taint step from `Argument[this]` to `ReturnValue`? IIRC we've defaulted to having small bit-width apis that are technically taint-propagating as taint steps and then rule them out with type-based barriers.